lib/product-import.coffee

```diff
@@ -448,23 +448,37 @@ class ProductImport
 
   _prepareUpdateProduct: (productToProcess, existingProduct) ->
     productToProcess = @_ensureDefaults(productToProcess)
+    if (productToProcess.state)
+      stateRef = productToProcess.state
+      statePredicate = "id=\"#{productToProcess.state.id}\" and type=\"ProductState\""
+    else
```

Comment:
As @butenkor suggested elsewhere, I think it would be good to implement a config option so we can insist on states for Marktplatz usage without foisting them on other users.  (Now I've just got to figure out how to do this via the way the sphere-product-importer is called for us via, apparently, the sphere-node-product-csv-sync.)